packages/twenty-front/src/modules/settings/data-model/fields/forms/phones/components/SettingsDataModelFieldPhonesForm.tsx

```diff
@@ -37,12 +41,13 @@ export const SettingsDataModelFieldPhonesForm = ({
     .sort((a, b) => a.countryName.localeCompare(b.countryName))
```

Comment:
```typescript
  const countries = [
    { label: 'No country', value: '' },
    ...useCountries()
      .sort((a, b) => a.countryName.localeCompare(b.countryName))
      .map((country) => ({
        label: `${country.countryName} (+${country.callingCode})`,
        value: country.countryCode,
      })),
  ];
```
then you don't need the unshift below. Wdyt?

Reply:
agreed